client/signup/steps/domains/index.jsx

```diff
@@ -290,16 +290,22 @@ class DomainsStep extends Component {
 		if ( ! isPurchasingItem ) {
 			switch ( flowName ) {
 				case 'onboarding':
-					loadExperimentAssignment(
-						'calypso_gf_signup_onboarding_free_free_dont_miss_out_modal_v3'
-					);
-					loadExperimentAssignment( 'calypso_onboarding_plans_paid_domain_on_free_plan' );
+					if ( isPurchasingItem ) {
```

Comment:
The condition above is checking for `! isPurchasingItem`, so there is no chance this condition works :) 

Also, could you explain the intention behind this change? That way we can better test and review the code.

Reply:
Thanks for raising this James! I have fixed it!

Reply:
> Also, could you explain the intention behind this change? That way we can better test and review the code.

Actually the conditions for preloading the experiment were incorrect. All experiments were pre loaded only when there was a free subdomain. But two of these experiments actually activated when there is a paid domain.